RealmBrowser/Views/RLMBasicTableCellView.m

```diff
@@ -32,9 +32,15 @@ - (instancetype)initWithFrame:(NSRect)frameRect
     textField.translatesAutoresizingMaskIntoConstraints = NO;
     textField.bordered = NO;
     textField.drawsBackground = NO;
-    textField.usesSingleLineMode = YES;
-    textField.lineBreakMode = NSLineBreakByTruncatingTail;
     
+    if ([textField respondsToSelector:@selector(setUsesSingleLineMode:)]) {
+        textField.usesSingleLineMode = YES;
```

Comment:
Sorry for the delay @bdash! I wanted to go back and investigate this some more before I replied.

I did some digging, and I discovered that `NSCell`, the superclass two levels above `NSTextFieldCell` has an identical property (i.e. `usesSingleLineMode`) that's available.

But that being said, the Browser doesn't seem to be using `NSCell`s at all. Every cell view we have in there is being subclassed from `NSTableViewCell` instead. So I'm not really sure what we'd have to do to get that functionality in our current setup.